### ISSUE TITLE
Demo Header Image Missing Alt Text

### ISSUE THREAD

**Issue by mgifford** [LiaScript/LiaScript#241]
Title: WCAG 1.1.1: Ensure <img> elements have alternative text or a role of none or presentation (img)
Tags: Accessibility, WCAG 1.1.1, image-alt

Issue: Ensure <img> elements have alternative text or a role of none or presentation (image-alt - https://accessibilityinsights.io/info-examples/web/image-alt)

Target application: Start by asking these types of questions: · Module 2 Sprint 3 - https://liascript.github.io/course/?https://raw.githubusercontent.com/usds/ditap-curriculum-update/main/3_Curriculum/3A_DITAP-Core-Curriculum/Module-2/Sprint-3/Self-Paced%20Materials/1_Defining%20Success%20for%20your%20Digital%20Services%20Acquisition.md#13

Element path: img

Snippet: <img src="icon.ico" class="lia_header__logo">

How to fix: 
Fix any of the following:
  Element does not have an alt attribute
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

Environment: Microsoft Edge version 139.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by andre-dietrich** [comment]
Thanks for pointing me into this direction ... I thought screen readers will ignore images with alt="" or without an alt-attribute at all ... I fixed this, this will be updated in the next release ...
